testsuite/features/upload_files/example.ks

```diff
@@ -19,9 +19,9 @@ keyboard us
 mouse none
 skipx
 
-# TODO pre script for eth1 and other complications?
+# root password is 'linux'
 network --device eth0 --bootproto dhcp
-#rootpw --iscrypted $1$mumble
+rootpw --iscrypted $6$LdF0Ddaf5q/wqYk6$x04erFOijr7U82EB2GL24Ko4yWvyVo4S91bg9Yp08PLDcLBwmxwJpfKox1vlZ/faFED.dbfAe5ofgoJtHCkia.
```

Comment:
What hash function did you use?

Reply:
If I recall correctly @nodeg, `$6$` means md5sum.

I did not use any hash function, I just copy-pasted from `/etc/shadow` :stuck_out_tongue: .